### PR TITLE
Make sure that the PassengerCategoryies and the PrivateMode are done in NeTEx, so that it works out.

### PR DESCRIPTION
**Pull request by ue71603**
fixes #349

### CODE REVIEW

**Comment by ue71603** [comment]
addresses #434 as well

File: xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd [hunk 1]
```diff
@@ -498,6 +498,31 @@ Rail transport, Roads and Road transport
...
+				<xsd:annotation>
+					<xsd:documentation>USER PROFILEs that are allowed on this line. Omitted means all.</xsd:documentation>
+				</xsd:annotation>
+				<xsd:complexType>
+					<xsd:sequence>
+						<xsd:element ref="UserProfileRef" minOccurs="0" maxOccurs="unbounded"/>
+					</xsd:sequence>
+				</xsd:complexType>
+			</xsd:element>
+			<xsd:element name="accessModes" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:511]: Please do the enumeration list you mentioned in the call.

**Comment by ue71603** [comment]
https://wiki.openstreetmap.org/wiki/Tag:bicycle%3Ddismount

**Comment by ue71603** [comment]
RequiredAccessModes

**Comment by ue71603** [comment]
different group so that is part of LineGroup

**Comment by ue71603** [comment]
create a group and add it also to ServiceJourney.

**Comment by ue71603** [comment]
Nick: Site has an element like that.

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Comment by nick-knowles** [comment]
There is a small dependency hygiene issue here that has been overlooked.
**_UserType_** enums are  in Part3   (being created as a concept).  **_Line_** in Part2, so this gives a forward  dependency  from Part2 on Part3, something we normally strictly avoid.

A  Straightforward  way to resolved would be to move the UserTypeEnum to the ServiceRestrictions package in Part1 which contains various values relevant for describing both fares and journeys

**Pull request merged**